ibm/resource_ibm_is_lb.go

```diff
@@ -722,6 +732,40 @@ func lbUpdate(d *schema.ResourceData, meta interface{}, id, name string, hasChan
 			return fmt.Errorf("Error Updating vpc Load Balancer : %s\n%s", err, response)
 		}
 	}
+
+	if hasChangedSecurityGroups {
+		if len(add) > 0 {
+			for _, d := range add {
+				createSecurityGroupTargetBindingOptions := &vpcv1.CreateSecurityGroupTargetBindingOptions{}
+				createSecurityGroupTargetBindingOptions.SecurityGroupID = &d
+				createSecurityGroupTargetBindingOptions.ID = &id
+				_, response, err := sess.CreateSecurityGroupTargetBinding(createSecurityGroupTargetBindingOptions)
+				if err != nil {
+					return fmt.Errorf("Error while creating Security Group Target Binding %s\n%s", err, response)
+				}
+			}
+		}
+		if len(remove) > 0 {
+			for _, d := range remove {
+				getSecurityGroupTargetOptions := &vpcv1.GetSecurityGroupTargetOptions{
+					SecurityGroupID: &d,
+					ID:              &id,
+				}
+				_, response, err := sess.GetSecurityGroupTarget(getSecurityGroupTargetOptions)
+				if err != nil {
+					if response != nil && response.StatusCode == 404 {
+						return nil
```

Comment:
Here should it be continue instead of return nil?
since we should check remaining elements

Reply:
@hkantare PR comments oncorporated , Thank you very much. Can you please check now :)